### PR TITLE
Warning when scheduling on appointment day

### PR DESCRIPTION
**Pull request by NeilvB**
Part two of [story](https://www.pivotaltracker.com/n/projects/2500129/stories/178480897)

Specifically regarding adding a warning message for when today is the appointment today (see [comment](https://www.pivotaltracker.com/story/show/178480897/comments/225606215))

#### UI

image

### CODE REVIEW

File: src/components/WorkOrder/AppointmentDetails.js [hunk 1]
```diff
@@ -41,36 +42,24 @@ const AppointmentDetails = ({ workOrder, schedulerSessionId }) => {
...
       return (
-        <Link href={href}>
-          <a className="lbh-link">{linkText}</a>
-        </Link>
+        <ScheduleInternalAppointmentLink
+          workOrderReference={workOrder.reference}
+          hasExistingAppointment={hasExistingAppointment}
+          appointmentIsToday={workOrder.appointmentIsToday()}
+        />
       )
```
Comment (EGiataganas) [inline src/components/WorkOrder/AppointmentDetails.js:63]: That's a great improvement moving them into separate components. I am wondering if it's worth to add another component `ScheduleAppointmentLink` which will have the if logic, so we have only one component here.

Comment (NeilvB) [reply src/components/WorkOrder/AppointmentDetails.js:63]: Thanks for this - I think this is a great shout but are you OK if I implement this in a separate PR another time? 

I'm keen to get this story over to QA as it has been delayed for a few different reasons.

**Review by siapankina — APPROVED** [review]
🚢

File: src/components/WorkOrder/AppointmentDetails.js (hunk 1 above)
Comment (siapankina) [reply src/components/WorkOrder/AppointmentDetails.js:63]: Agree with Evangelos, and it is fine by me if we implement it in a separate PR

Comment (EGiataganas) [reply src/components/WorkOrder/AppointmentDetails.js:63]: Yeah sure, we can pick up this on a separate PR.

**Review by EGiataganas — APPROVED** [review]

**Review by EGiataganas — APPROVED** [review]

**Review by EGiataganas — APPROVED** [review]

**Pull request merged**